packages/hoverable/babel.config.js

```diff
@@ -1 +1,9 @@
-module.exports = require('../shared/babel.config');
+const sharedBabelConfig = require('../shared/babel.config');
```

Comment:
No that we have a correct preset for jest, I guess we can remove this from Babel?